src/overview/overview.md

```diff
@@ -15,7 +15,21 @@ ___
 
 General presentation of *what is the Substra framework*.
 
-Substra is a framework offering secure and trustless traceability for distributed orchestration of machine learning tasks among partners. It aims at being compatible with privacy-enhancing technologies to complement their use to provide efficient and transparent privacy-preserving workflows for data science. Its ambition is to make new scientific and economic data science collaborations possible.
+Substra is a framework offering distributed orchestration of machine learning tasks among partners while guaranteeing secure and trustless traceability of all operations.
+
+Data scientist:
+
+- Use your own ML algorithm with any Python ML framework
+- Ship your algorithm on remote data for training and/or prediction
+- Build advanced Federated Learning strategies for learning accross several remote datasets
+
+Data provider:
```

Comment:
I would call them `Data controllers` rather than `data provider`, because medical doctors don't like the term `data provider`

Reply:
Ok, noted, thank you for this input!